crates/rpc/rpc/Cargo.toml

```diff
@@ -34,6 +34,8 @@ revm = { workspace = true, features = [
     "optional_block_gas_limit",
     "optional_eip3607",
     "optional_no_base_fee",
+    "optional_balance_check",
+    "optional_gas_refund"
```

Comment:
why do we need these?

should they only be enabled if optimism feature is enabled?

Reply:
Good spot- we'll want to remove these, left-over from before we added the state transition changes to `revm`.